Solaris provides sendmmsg as of 11.3.32
It was added at the same time as MSG_WAITFORONE

Patch shared upstream: https://github.com/libuv/libuv/issues/4715
diff --git a/src/unix/udp.c b/src/unix/udp.c
index 67c01f7d..31cbde5f 100644
--- a/src/unix/udp.c
+++ b/src/unix/udp.c
@@ -1298,7 +1298,8 @@ static int uv__udp_sendmsgv(int fd,
   r = 0;
   nsent = 0;
 
-#if defined(__linux__) || defined(__FreeBSD__) || defined(__APPLE__)
+#if defined(__linux__) || defined(__FreeBSD__) || defined(__APPLE__) || \
+  (defined(__sun__) && defined(MSG_WAITFORONE))
   if (count > 1) {
     for (i = 0; i < count; /*empty*/) {
       struct mmsghdr m[20];
